ci/update_securityhub_suppressions/suppressions.yml

```diff
@@ -242,6 +242,32 @@ Suppressions:
         - arn:aws:iam::776473272850:user/dwp-dev
       notes: MFA is not enabled as this user is only a service account for access to SQS
 
+  ## Lambda
+  "lambda-dlq-check-conformance-pack-1dpd692bc":
```

Comment:
Are these new to this conformance pack then, or is there an old version we should remove?

Reply:
Discussed offline, this is a new suppression rather than related to a change in conformance pack version